src/common/git/getChangedFileLines.ts

```diff
@@ -1,23 +1,27 @@
 import { exec } from "child_process";
 
-import { getGitHubEnvVariables } from "../../config";
+import { getGitHubEnvVariables, getGitLabEnvVariables } from "../../config";
+import { GITHUB, GITLAB } from "../constants";
 
-export const getChangesFileLinesCommand = (
-  isCi: boolean,
+export const getChangesFileLinesCommand = async (
+  isCi: string,
   fileName: string
-): string => {
-  if (isCi) {
+): Promise<string> => {
+  if (isCi === GITHUB) {
     const { githubSha, baseSha } = getGitHubEnvVariables();
     return `git diff -U0 --diff-filter=AMT ${baseSha} ${githubSha} ${fileName}`;
+  } else if (isCi === GITLAB) {
+    const { gitlabSha, mergeRequestBaseSha } = await getGitLabEnvVariables();
```

Comment:
Is there a reason why you are awaiting getGitlabVariables() should be sync no?

Reply:
Yeah defs, must be a artifact from when I was testing things out